scripts/update_package_json_files.js

```diff
@@ -0,0 +1,98 @@
+const execa = require('execa')
+const path = require('path')
+const {writeJsonToPackageJson, getPackageJsonContent } = require('./util')
+
+function bumpVersionInVSCodeRepo({ version, name, displayName, description, preview }) {
+  const vscodePackageJsonPath = path.join(__dirname, '../packages/vscode/package.json')
+  let content = getPackageJsonContent({path: vscodePackageJsonPath})
+  content['version'] = version
+  content['name'] = name
+  content['displayName'] = displayName
+  content['description'] = description
+  content['preview'] = preview
+  writeJsonToPackageJson({content: content, path: vscodePackageJsonPath})
+}
+
+function bumpLSPVersionInExtension({version}) {
+  const vscodePackageJsonPath = path.join(__dirname, '../packages/vscode/package.json')
+  let content = getPackageJsonContent({path: vscodePackageJsonPath})
+  content['dependencies']['@prisma/language-server'] = version
+  writeJsonToPackageJson({content: content, path: vscodePackageJsonPath})
+}
+
+function bumpVersionsInRepo({ channel, newExtensionVersion, newPrismaVersion = '' }) {
+  const languageServerPackageJsonPath = path.join(__dirname, '../packages/vscode/package.json')
+  const rootPackageJsonPath = path.join(__dirname, '../package.json')
+
+  // update version in packages/vscode folder
+  if (channel === 'dev' || channel === 'patch-dev') {
+    // change name, displayName, description and preview flag to Insider extension
+    bumpVersionInVSCodeRepo({
+      version: newExtensionVersion,
+      name: "prisma-insider",
+      displayName: "Prisma - Insider",
+      description: "This is the Insider Build of the Prisma VSCode extension (only use it if you are also using the dev version of the CLI.",
+      preview: true
+    })
+  } else {
+    bumpVersionInVSCodeRepo({
+      version: newExtensionVersion,
+      name: "prisma",
+      displayName: "Prisma",
+      description: "Adds syntax highlighting, formatting, auto-completion, jump-to-definition and linting for .prisma files.",
+      preview: false
+    })
+  }
+
+  // update Prisma CLI version in packages/language-server folder
+  if (newPrismaVersion !== '') {
+    (async () => {
+      const { stdout } = await execa('npx', ['-q', `@prisma/cli@2.6.0`, 'version', '--json']);
```

Comment:
Is it intentional that this version is hard coded?

Reply:
No it is not 😅 Leftover from debugging this. Will change this.